### DISCUSSION TITLE
Select/Reference Admob interstitial banner close button

### DISCUSSION THREAD

**Discussion by apaolino** [leancodepl/patrol#845] [Q&A]
Hi,
using Patrol 0.18.8 and I'd like to programmatically close an Admob interstitial banner within my integration tests for both Android and iOS. Do you know how I can achieve it?
I've tried with `Selector(contentDescriptionContains: "close")` but no luck.

**Comment by bartekpacia** [comment]
Hi @apaolino,

I can't help you without more details. Please share some screenshots, code, and [perform view hierarchy dump](https://patrol.leancode.co/tips-and-tricks#inspecting-native-view-hierarchy).
